draft-js-counter-plugin/src/CharCounter/index.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component, PropTypes } from 'react';
+import unionClassNames from 'union-class-names';
+import { Map } from 'immutable';
+
+class CharCounter extends Component {
+
+  static propTypes = {
+    editorState: PropTypes.any.isRequired,
+    theme: PropTypes.any,
+  };
+
+  getCharCount(editorState) {
+    const plainText = editorState.getCurrentContent().getPlainText('');
+    const cleanString = plainText.replace(/(?:\r\n|\r|\n)/g, '').trim();
```

Comment:
Can you comment on every part of the regex to make it clear which parts are there & why?